src/qml/filters/audio_lowpass/ui.qml

```diff
@@ -21,31 +21,52 @@ import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import Shotcut.Controls 1.0 as Shotcut
 
-Item {
+Shotcut.KeyframableFilter {
+    property string cutoffProperty: '0'
+    property string stagesProperty: '1'
+    property string wetnessProperty: 'wetness'
     width: 350
     height: 125
+    keyframableParameters: preset.parameters
+    startValues: [0, 0, 0]
+    middleValues: [0, 0, 0]
```

Comment:
The middle value for wetness should be 1 to allow simple keyframes to fade in and/or out wetness by default.

Reply:
Should be good now!